geode-core/src/test/java/org/apache/geode/cache/client/internal/OpExecutorImplJUnitTest.java

```diff
@@ -99,7 +99,7 @@ public RuntimeException generateCancelledException(Throwable e) {
   @Test
   public void testExecute() throws Exception {
     OpExecutorImpl exec = new OpExecutorImpl(manager, queueManager, endpointManager, riTracker, 3,
-        10, cancelCriterion, null);
+        10, -1, cancelCriterion, null);
```

Comment:
When the default value for the timeout is being used, instead of hard-coding the value like here, it would be better to use a reference to the `DEFAULT_SERVER_CONNECTION_TIMEOUT` defined in PoolFactory.java.